### ISSUE TITLE
Binary files (documentation no longer accurate)

### ISSUE THREAD

**Issue by fauxmight** [Mic92/sops-nix#601]
The documentation reads as follows regarding handling binary files:

```sh
$ cp /etc/krb5/krb5.keytab > krb5.keytab
$ sops -e -o krb5.keytab
# an example of what this might result in:
$ head krb5.keytab
```

The `-o` flag is not usable. (Yields "flag provided but not defined: -o")

Also, the first line appears that either the command should be `cat` or the redirect should be eliminated. This line should probably read either
```sh
$ cp /etc/krb5/krb5.keytab krb5.keytab
```
or
```sh
$ cat /etc/krb5/krb5.keytab > krb5.keytab
```

Is it still possible to process binary files?

**Comment by sedlund** [comment]
sops has recently released 3.9.0 that has updated their options - you should refer to their project for the [documentation](https://getsops.io/docs/) .

`-o` is deprecated 

if you do not have rules setup in your `.sops.yaml` for the specified file, you'll probably need to use `--input-type` and `--output-type` appropriately.

The documentation here should be updated to reflect that.  If you have more changes I'm sure they would be welcome.

**Comment by fauxmight** [comment]
@sedlund: Thank you for the prompt, well-informed reply.
I generated a minor pull request with the documentation changes and am closing this issue.
